src/pages/home/HomeBlogPreview.jsx

```diff
@@ -82,7 +83,7 @@ function ReadMore() {
   );
 }
 
-function DesktopBlogPreview({ featuredPosts, allPosts }) {
+function DesktopBlogPreview({ FirstPost, allPosts }) {
```

Comment:
```suggestion
function DesktopBlogPreview({ allPosts }) {
  const firstPost = allPosts?.slice(0, 1);
```
...then reformat this like so...